modules/cluster/src/test/java/io/fluo/cluster/runner/MiniAppRunnerIT.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2015 Fluo authors (see AUTHORS)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fluo.cluster.runner;
+
+import io.fluo.core.ITBaseImpl;
+import io.fluo.core.TestTransaction;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class MiniAppRunnerIT extends ITBaseImpl {
+
+  @Test
+  public void testSimple() {
+    TestTransaction tx1 = new TestTransaction(env);
+    tx1.mutate().row("row1").fam("fam1").qual("q1").set("v1");
```

Comment:
It doesn't create notificaitons.   Do you think I should create them?  They will probably be processed by the time `runner.waitUntilFinished()` is called.